python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -2077,3 +2077,82 @@ def convert_sqrt(node, **kwargs):
         name=name,
     )
     return [node]
+
+@mx_op.register("square")
+def convert_square(node, **kwargs):
+    """Map MXNet's square operator attributes to onnx's Pow operator
+    and return the created node.
+    """
+    onnx = import_onnx_modules()
+    name = node["name"]
+    proc_nodes = kwargs["proc_nodes"]
+    inputs = node["inputs"]
+
+    input_node_a_id = kwargs["index_lookup"][inputs[0][0]]
+
```

Comment:
nit: in general feel that there are a lot of unnecessary blank lines in this method. also you can remove line 2096 have `np.array([2])` directly. but this change is not a blocker